company.rb

```diff
@@ -0,0 +1,139 @@
+class Company
+  @@count = 0
+  def initialize(comp_name, division)
+    @comp_name = comp_name
+    #@no_emp = no_emp
+    @division = division
+    @@count += 1
+  end
+  attr_accessor :comp_name, :division
+  def print_company_details
+    puts comp_name
+    #puts no_emp
+    puts division
+    puts @@count
+  end
+
+  def read_file
+    fileobject = File.new("hello world", "r")
+    puts fileobject.read
+  end
+end
+
+class Product < Company
```

Comment:
Is-A relationship: Whenever one class inherits another class, it is called an IS-A relationship.
Do you think the product is a company?